api/app/Models/ArticleMeta.php

```diff
@@ -21,9 +21,7 @@ class ArticleMeta extends BaseModel
      */
     public $table = 'article_metas';
 
-    protected $primaryKey = 'meta_name';
-
-    protected $fillable = ['meta_name', 'meta_content'];
+    protected $fillable = ['id', 'article_id', 'meta_name', 'meta_content'];
```

Comment:
The primary key for all entities must match the route segment + `_id` so in this case it would be `meta_id`. See the other models for the pattern.